_examples/sql/event-log/README.md

```diff
@@ -0,0 +1,51 @@
+# Event log
+
+This example shows how to use the SQL Publisher from [SQL Pub/Sub](https://github.com/ThreeDotsLabs/watermill-sql).
+
+## Background
+
+Some PubSubs (e.g. Kafka) come with support for storing processed messages, possibly even with no expiration date.
+This can be useful for audit purposes or to reply selected messages again in the future. But what if you'd like to use
+a PubSub that offers no storage and an event log is needed?
+
```

Comment:
maybe also info that it's useful when you don't have any pubsub infrastructue, but just SQL?

Reply:
I'll add that one to the general SQL docs, as this example is focused on a particular problem.

We could also add a third example using just the SQL pub/sub, but there's already something like this in the getting started section, so I feel that would be overkill.